uninstall/uninstall.sh

```diff
@@ -6,4 +6,196 @@ APP_PATH=`cd "$APP_PATH"; pwd`
 HOME_PATH=~
 INSTALL_PATH="${HOME_PATH}/.splashkit"
 
-rm -rf ${INSTALL_PATH}
+SKM_PATH=`cd "$APP_PATH/.."; pwd`
+
+HAS_PYTHON3=false
+
+source "${SKM_PATH}/tools/set_sk_env_vars.sh"
+
+echo "Attempting to uninstall SplashKit libraries"
+echo "This may require root access, please enter your password when prompted."
+echo
+
+if [ "$IS_WINDOWS" = true ]; then
+    PRIVILEGED=""
+else
+    PRIVILEGED="sudo"
+fi
+
+# Get directories for each OS
+echo "Detecting operating system..."
+if [ "$SK_OS" = "macos" ]; then
+    LIB_FILE="/usr/local/lib/libSplashKit.dylib"
+    LIB_DEST="/usr/local/lib"
+    INC_DEST="/usr/local/include"
+
+    # Check if python3 installed on macOS
+    if command -v python3 &> /dev/null && command -v brew &> /dev/null; then
+        HAS_PYTHON3=true
+    fi
+elif [ "$SK_OS" = "linux" ]; then
+    LIB_FILE="/usr/local/lib/libSplashKit.so"
+    LIB_DEST="/usr/local/lib"
+    INC_DEST="/usr/local/include"
+
+    # Check if python3 installed on Linux/WSL
+    if command -v python3 &> /dev/null; then
+        HAS_PYTHON3=true
+    fi
+elif [ "$SK_OS" = "win64" ]; then
+    # WIN_OUT_DIR="${WINDIR}/System32"
+    if [ "$MSYSTEM" = "MINGW64" ]; then
+        LIB_FILE="${SKM_PATH}/lib/win64/SplashKit.dll"
+        LIB_DEST="/mingw64/lib"
+        INC_DEST="/mingw64/include"
+    elif [ "$MSYSTEM" = "CLANG64" ]; then
+        LIB_FILE="${SKM_PATH}/lib/win64/SplashKit.dll"
+        LIB_DEST="/clang64/lib"
+        INC_DEST="/clang64/include"
+    # elif [ "$MSYSTEM" = "CLANGARM64" ]; then
+    #     LIB_FILE="${SKM_PATH}/lib/win64/SplashKit.dll"
```

Comment:
We would need an arm version of the dll here... so a separate folder?

Reply:
Yes, maybe "winARM64" or similar? (When we get to that)

I have commented these lines out for now, but can update the CLANGARM64 lib path to use a different name in the comment if you'd like